### PR TITLE
Update ghactions

### PR DESCRIPTION
**Pull request by shammowla**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have made any necessary test changes and all tests pass.
- [ ] I have run the Sandbox successfully.
- [ ] I've updated the schema in extension.json or no changes are necessary.
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by shammowla** [comment]
1) Add missing parcel dependency required by the functional tests.
2) Update Testcafe version.
3) Update the github actions to latest extension release.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -17,28 +17,28 @@ jobs:
...
-        tunnelName: $GITHUB_JOB
-    - name: Run Unit Test
-      run: npx karma start karma.saucelabs.conf.js --single-run
-      env:
-        BUILD_NUMBER: $GITHUB_RUN_NUMBER
-        BUILD_ID: $GITHUB_RUN_ID
-        JOB_NUMBER: $GITHUB_JOB
+      - uses: actions/checkout@v2.4.2
+#      - uses: actions/cache@v2.1.4
+#        id: npm-cache
```
Comment (carterworks) [inline .github/workflows/tests.yml:22]: (non-blocking nitpick) delete this commented-out configuration

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -17,28 +17,28 @@ jobs:
...
-        username: ${{ secrets.SAUCE_USERNAME }}
-        accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
-        tunnelName: $GITHUB_JOB
-    - name: Run Unit Test
-      run: npx karma start karma.saucelabs.conf.js --single-run
-      env:
-        BUILD_NUMBER: $GITHUB_RUN_NUMBER
-        BUILD_ID: $GITHUB_RUN_ID
-        JOB_NUMBER: $GITHUB_JOB
+      - uses: actions/checkout@v2.4.2
```
Comment (carterworks) [inline .github/workflows/tests.yml:20]: (non-blocking nitpick) I would prefer to see this pinned to just v2. As-is, it requires manual work on our part to update the action.

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -55,11 +55,8 @@ jobs:
           ]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '18'
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v2.4.2
+      - uses: actions/cache@v2.1.4
```
Comment (carterworks) [inline .github/workflows/tests.yml:59]: (non-blocking nitpick) Same-I'd prefer these to require v2 instead of specific versions.

**Review by carterworks — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 3 above)
Comment (shammowla) [reply .github/workflows/tests.yml:54]: Made the suggested changes.

**Pull request merged**